listeners.py

```diff
@@ -33,13 +34,19 @@ def process_timeout(self, delta_seconds):
 class StatisticsWatcher(BaseWatcher):
     def __init__(self, get_logfile):
         self.stats = dict()
+        self.rss_results = sampler.rss_summary()
         self.failed_tasks = []
         self.get_logfile = get_logfile
+        self.long_tasks = []
```

Comment:
We don't need an order here, so it seems logical to define it as `set()`.

Reply:
Sure, changed.

Reply:
Ok, changed.